bagit.py

```diff
@@ -388,7 +392,7 @@ def _load_manifests(self):
             alg = os.path.basename(manifest_file).replace(search, "").replace(".txt", "")
             self.algs.append(alg)
 
-            with open(manifest_file, 'r') as manifest_file:
+            with open_text_file(manifest_file, 'r', encoding='utf-8-sig') as manifest_file:
```

Comment:
Oh, thanks, I'd forgotten that we allow that. Something to bookmark for the spec updates since we don't mention which standard the encoding name values should be matched against.